src/arche/rules/price.py

```diff
@@ -74,13 +76,13 @@ def compare_prices_for_same_urls(
         missing and new `product_url_field` tagged fields.
     """
     result = Result("Compare Prices For Same Urls")
-    url_field = tagged_fields.get("product_url_field")
+    url_field: Any = tagged_fields.get("product_url_field")
```

Comment:
Optional[str], same for all `Any` below.
https://docs.python.org/3/library/typing.html#typing.Any
https://docs.python.org/3/library/typing.html#typing.Optional